test/parallel/test-fs-read-empty-buffer.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+require('../common');
+const fixtures = require('../common/fixtures');
+const assert = require('assert');
+const fs = require('fs');
+const filepath = fixtures.path('testcodes.txt');
```

Comment:
This and the additional fixtures should be unnecessary, we can use any existing fixtures for this test, like `fixtures.path('elipses.txt')`